apps/deploy-web/src/components/billing-usage/AccountOverview/AccountOverview.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { useCallback, useEffect, useMemo, useState } from "react";
+import { FormattedNumber } from "react-intl";
+import { Button, Card, CardContent, CardFooter, CardHeader, CardTitle, Snackbar, Switch } from "@akashnetwork/ui/components";
+import {} from "@akashnetwork/ui/components";
+import { LinearProgress } from "@mui/material";
+import { Plus } from "iconoir-react";
+import { useSnackbar } from "notistack";
+
+import { PaymentPopup } from "@src/components/billing-usage/PaymentPopup/PaymentPopup";
+import Layout from "@src/components/layout/Layout";
+import { Title } from "@src/components/shared/Title";
+import { PaymentSuccessAnimation } from "@src/components/user/payment/PaymentSuccessAnimation";
+import { useWalletBalance } from "@src/hooks/useWalletBalance";
+import { usePaymentMethodsQuery, useWalletSettingsMutations, useWalletSettingsQuery } from "@src/queries";
+
+export const AccountOverview: React.FunctionComponent = () => {
+  const [selectedPaymentMethodId, setSelectedPaymentMethodId] = useState<string | undefined>();
+  const [showPaymentPopup, setShowPaymentPopup] = useState(false);
+  const [showPaymentSuccess, setShowPaymentSuccess] = useState<{ amount: string; show: boolean }>({ amount: "", show: false });
+  const { enqueueSnackbar } = useSnackbar();
+  const { data: paymentMethods = [], isLoading: isLoadingPaymentMethods } = usePaymentMethodsQuery();
```

Comment:
it seems all payment methods are not actually needed here. If this is the case there is an endpoint for `GET /v1/payment-methods/default`